package.json

```diff
@@ -1,88 +1,85 @@
 {
   "name": "@4c/layout",
   "version": "0.2.1",
+  "files": [
+    "lib",
```

Comment:
there was so much junk on npm lol

Reply:
we should add it on the base cli

Reply:
i think we generally want to publish from `/lib` in many cases, though